api/pkg/backend/service.go

```diff
@@ -447,3 +446,133 @@ func (s *APIService) listStorageType(ctx context.Context, request *restful.Reque
 	storageTypes, err := s.backendClient.ListType(ctx, listTypeRequest)
 	return storageTypes, err
 }
+
+func (s *APIService) CreateTier(request *restful.Request, response *restful.Response) {
```

Comment:
Somewthere it should validate, the backends whether they really exists or not. Before Creating or adding to DB.

Reply:
done